packages/fxa-content-server/app/tests/spec/lib/email-domain-validator.js

```diff
@@ -134,7 +134,21 @@ describe('lib/email-domain-validator', () => {
       xhr.ajax.restore();
     });
   });
-
+  it('should resolve on a successful skip record', () => {
+    sinon.stub(xhr, 'ajax').returns(Promise.resolve({ result: 'skip' }));
+    $el.val('quuz@great.skip.co.gd');
+    return checkEmailDomain($el, view).then(() => {
+      assert.isTrue(view.logEvent.calledWith('emailDomainValidation.skip'));
+      assert.isTrue(view.logEvent.calledWith('emailDomainValidation.skip'));
+      assert.isTrue(
+        view.logFlowEvent.calledWith('email-domain-validation.skipped')
+      );
+      assert.isTrue(
+        view.logFlowEvent.calledWith('email-domain-validation.skip')
+      );
+      xhr.ajax.restore();
+    });
+  });
```

Comment:
Since we do nothing but `resolve()`, let's add a `catch` and call `assert.fail` in it, since it should not have rejected.